lib/iex/lib/iex/evaluator.ex

```diff
@@ -97,25 +97,33 @@ defmodule IEx.Evaluator do
     end
   end
 
-  defp do_eval(@break_trigger, state=%IEx.State{cache: ''}) do
+  defp do_eval(@break_trigger, state=%IEx.State{cache: ''}, _) do
     state
   end
 
-  defp do_eval(@break_trigger, state) do
+  defp do_eval(@break_trigger, state, _) do
     :elixir_errors.parse_error(state.counter, "iex", "incomplete expression", "")
   end
 
-  defp do_eval(latest_input, state) do
+  @history_key :__history_proc__
```

Comment:
Ok then, one last change: can we change this to `:iex_history_pid`? Prefixing it with iex is going to make it clearer what it is about if someone is inspecting it.